x/bank/autocli.go

```diff
@@ -87,5 +87,34 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 				},
 			},
 		},
+		Tx: &autocliv1.ServiceCommandDescriptor{
+			Service:              bankv1beta1.Msg_ServiceDesc.ServiceName,
+			EnhanceCustomCommand: true,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod: "Send",
+					Use:       "send [from_key_or_address] [to_address] [amount]",
+					Short:     "Send funds from one account to another.",
+					Long: `Send funds from one account to another.
```

Comment:
Let's precise that multi coins are supported and it should be separated by a space.